api/tests/unit/domain/events/handle-certification-rescoring_test.js

```diff
@@ -40,17 +40,15 @@ describe('Unit | Domain | Events | handle-certification-rescoring', function ()
       .resolves(certificationAssessment);
     certificationCourseRepository.get.withArgs(certificationCourse.getId()).resolves(certificationCourse);
 
-    const competenceMarkData2 = domainBuilder.buildCompetenceMark();
-    const competenceMarkData1 = domainBuilder.buildCompetenceMark();
-    const nbPix = Symbol('nbPix');
-    const status = Symbol('status');
-    const certificationAssessmentScore = {
-      nbPix,
-      status,
+    const competenceMarkData2 = domainBuilder.buildCompetenceMark({ score: 5 });
```

Comment:
*nitpick*
`Data` ? C'est un objet du domaine, non ?
```suggestion
    const competenceMark2 = domainBuilder.buildCompetenceMark({ score: 5 });
```

Reply:
c'est très vrai, merci